superset-frontend/src/SqlLab/components/SqlEditor/index.jsx

```diff
@@ -365,9 +373,15 @@ class SqlEditor extends React.PureComponent {
   }
 
   handleToggleAutocompleteEnabled = () => {
-    this.setState(prevState => ({
-      autocompleteEnabled: !prevState.autocompleteEnabled,
-    }));
+    this.setState(prevState => {
+      setInLocalStorage(
+        SQL_LAB_AUTOCOMPLETE_SETTING_KEY,
+        !prevState.autocompleteEnabled,
+      );
+      return {
```

Comment:
Why is a return necessary when it wasn’t previously?

Reply:
It does previously. An arrow function with just expression will use the value of that expression as the return value.